### DISCUSSION TITLE
[2021.08.25] 최솟값 구하기

### DISCUSSION THREAD

**Discussion by leechang4** [pyunji/Hyundai-ITnE-Team_QuadCore#42] [SQL]
[최솟값 구하기](https://programmers.co.kr/learn/courses/30/lessons/59038)

**Comment by pyunji** [comment]
[최솟값 구하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59038.sql)

**Comment by hyk339** [comment]
[최솟값 구하기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/f8f63afd3e440deee3f47536086285eada896fb8)

**Comment by leechang4** [comment]
[최솟값 구하기](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059038.sql)

**Comment by chaeyeon4u** [comment]
[최솟값 구하기](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59038.sql)
